File: test/fixtures/python/pkg/cpf.py
import warnings
from typing import Optional


def is_valid(cpf: str) -> bool:
    return len(cpf) == 11


def format_cpf(cpf: str, *, pad: bool = False) -> Optional[str]:
    return cpf


def validate(cpf):
    warnings.warn("use is_valid", DeprecationWarning)
    return is_valid(cpf)


def _helper(x):
    return x


def generate(count: int = 1, *more: str) -> list[str]:
    return []

File: test/fixtures/dotnet/Lib/Cpf.fs
module Lib.Cpf

open System

let private helper x = x

// let Commented x = x
let IsValid cpf =
    helper cpf <> ""

let Format (cpf: string) : string option =
    Some cpf

[<Obsolete("use IsValid")>]
let Validate cpf = IsValid cpf

let Generate () =
    "00000000000"

let constantValue = 42

module Nested =
    let Inner (a: int, b: int) = a + b
